stack.yaml

```diff
@@ -1,7 +1,24 @@
 resolver: lts-13.28
+packages:
+- .
+
+# For local dev
+# - ../pantry
+# - ../casa/casa-client
+# - ../casa/casa-types
+
 extra-deps:
 - http-download-0.1.0.0
-- pantry-0.1.1.0
 - rio-prettyprint-0.1.0.0
 - ansi-terminal-0.9.1
 - rio-0.1.10.0
+
+#- pantry-0.1.1.0
+- git: https://github.com/commercialhaskell/pantry.git
```

Comment:
Minor point, but I'd recommend using `github: commercialhaskell/pantry` instead, which can more cheaply download a tarball instead of cloning a Git repo. Same for `casa`.

Reply:
Cool, good to know!